events/templates/vote/meeting_vote.html

```diff
@@ -0,0 +1,70 @@
+{% extends "base.html" %}
+
+{% block head-content %}
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css">
+
+    {% load static %}
+    <link href="{% static 'vote/meeting_vote.css' %}" rel="stylesheet">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <style>
+    #remove {
```

Comment:
can you please move this to the css file?

Reply:
yes. Thanks 😄